.travis.yml

```diff
@@ -9,6 +9,10 @@ env:
           testflags="--test-type integ --pytest-args='-x'"
           use_codecov=true
           use_coveralls=false
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+          testflags="--test-type integ_nm --pytest-args='-x'"
+          use_codecov=true
```

Comment:
Codecov does not handle multiple uploads from one job that good IIRC. Something is strange with the diff in this PR, it shows the timeout change as diff:
https://codecov.io/gh/nmstate/nmstate/pull/748/diff

Reply:
I expected this may be a problem. I do not have a solution for this.